fontbe/src/features.rs

```diff
@@ -123,12 +123,12 @@ impl<'a> FeaVariationInfo<'a> {
 //except they have slightly different inputs?
 pub(crate) fn resolve_variable_metric(
     static_metadata: &StaticMetadata,
-    values: &BTreeMap<NormalizedLocation, OrderedFloat<f32>>,
+    values: impl IntoIterator<Item = (NormalizedLocation, OrderedFloat<f32>)>,
```

Comment:
this reminds me that another potential speedup could be memoizing the results of this fn call, I think we call it with a lot of redundant values and it's expensive.

Also possibly a better signature would be just `impl Iterator`, since this signature requires ownership which means an additional clone at the call site?

Reply:
+1 to impl Iterator, will change